MessengerOnFileForLinux/scripts/messenger_binar

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```

Comment:
Rozumiem, że to jest do testów. czy to powinno być w script a nie raczej w folderze z odpowiednimi utekami? Tak to ktoś może coś pomylić patrząc po nazwach.

Reply:
ok, przerzuce